package.json

```diff
@@ -12,16 +12,17 @@
     "repository": "https://github.com/inclusive-design/forgiving-data",
     "dependencies": {
         "infusion": "3.0.0-dev.20210120T204128Z.6e4be079f.FLUID-6580",
-        "kettle": "2.1.0",
+        "kettle": "2.2.0",
         "rimraf": "3.0.2",
         "json5": "2.2.0",
-        "simple-git": "2.35.2",
-        "papaparse": "5.3.0"
+        "data-update-github": "0.1.0",
```

Comment:
The `data-update-github` repo has been renamed to `git-ops-api` and the latest version is "1.0.0". Please adjust its usage in this repo. Thanks.